requirements/constraints.txt

```diff
@@ -11,9 +11,11 @@
 # Common constraints for Open edX repos
 -c https://raw.githubusercontent.com/openedx/edx-lint/master/edx_lint/files/common_constraints.txt
 
-# Keep same platform version
-pylint<3.0
-pycodestyle<2.9.0
+# Versions >= 3.3.0 drop support for Python 3.8
+# Ensures a consistent pylint version across all Python environments to avoid triggering varying warnings
+pylint<3.3.0
+
+# Higher version has breaking changes in the platform
```

Comment:
Could we give a little more detail as it is here? https://github.com/openedx/edx-platform/blob/31d170a687795a7bda1e7b510d03d08fd00e1f22/requirements/constraints.txt#L48-L52

Reply:
Done !